README.md

```diff
@@ -1,2 +1,51 @@
 # eigen3_cmake_module
-Adds a custom find module for Eigen3
+
+This package adds a [CMake find module](https://cmake.org/cmake/help/v3.14/manual/cmake-developer.7.html#find-modulesjj) for [Eigen3](https://eigen.tuxfamily.org/dox/) that sets [standard cmake variables](https://cmake.org/cmake/help/v3.5/manual/cmake-developer.7.html#standard-variable-names).
```

Comment:
Nitpick: proper name: standard cmake variables -> standard CMake variables

Reply:
`CMake` instead of `cmake` in 3841fd71e0447cea2b83334550140b0abf10c4af